### PR TITLE
feat(wallet) deliver pending changed events to clients

### PR DESCRIPTION
**Pull request by stefandunca**
### Updates status-desktop [#11233](https://github.com/status-im/status-desktop/issues/11233)

Add wallet events feed to TransactionManager and send pending changed events on add and delete
    
Also

- Remove hardcoded values in the filter query
- Small improvement in query

### CODE REVIEW

File: services/wallet/transfer/transaction.go [hunk 1]
```diff
@@ -252,11 +260,30 @@ func (tm *TransactionManager) AddPending(transaction PendingTransaction) error {
...
+			Accounts: addresses,
+			At:       int64(timestamp),
+		})
+	}
+}
+
 func (tm *TransactionManager) DeletePending(chainID uint64, hash common.Hash) error {
 	_, err := tm.db.Exec(`DELETE FROM pending_transactions WHERE network_id = ? AND hash = ?`, chainID, hash)
+	if err == nil {
+		tm.notifyPendingTransactionListeners(chainID, nil, 0)
```
Comment (dlipicar) [inline services/wallet/transfer/transaction.go]: shouldn't we get the from/to addresses from the deleted transaction to put them in the notification?

File: services/wallet/transfer/transaction.go [hunk 2]
```diff
@@ -21,31 +22,38 @@ import (
 	"github.com/status-im/status-go/services/wallet/bigint"
 	"github.com/status-im/status-go/services/wallet/bridge"
 	wallet_common "github.com/status-im/status-go/services/wallet/common"
+	"github.com/status-im/status-go/services/wallet/walletevent"
 	"github.com/status-im/status-go/transactions"
 )
 
+const (
+	// EventNewTransfers emitted when new block was added to the same canonical chan.
```
Comment (dlipicar) [inline services/wallet/transfer/transaction.go]: typo `canonical chan`

**Review by dlipicar — APPROVED** [review]

File: services/wallet/transfer/transaction.go (hunk 1 above)
Comment (stefandunca) [reply services/wallet/transfer/transaction.go]: I intentionally left out this optimization given that this should be temporary and improve with the implementation of dynamic updates in the future.

Comment (stefandunca) [reply services/wallet/transfer/transaction.go]: I followed your advice in the end; thanks for the heads up. But also found that the to_address in pending is wrong. I will follow up on that issue.

**Pull request merged**